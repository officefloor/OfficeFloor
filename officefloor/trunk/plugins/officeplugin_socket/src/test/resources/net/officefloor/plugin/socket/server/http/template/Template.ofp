<html>
<body>
Template ${TemplateName}
<br />
<!-- [NullBean] -->
Should be no template name ${TemplateName} available.
<br />
with a listing of <a href="#{beans}">beans</a>:
<br />
<table>
    <tr>
        <td>Name</td>
        <td>Description</td>
    </tr>
    <!-- [List] -->
    <tr>
        <td>${Name}</td>
        <td>${description}</td>
    </tr>
    <!-- [Tail] -->
</table>
<form action="#{submit}">
    <input type="submit" />
</form>
</body>
</html>