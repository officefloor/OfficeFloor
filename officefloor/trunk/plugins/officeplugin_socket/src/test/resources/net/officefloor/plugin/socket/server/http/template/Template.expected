<html>
<body>
Template Test
<br />

Should be no template name  available.
<br />
with a listing of <a href="/WORK/beans.task">beans</a>:
<br />
<table>
    <tr>
        <td>Name</td>
        <td>Description</td>
    </tr>
    
    <tr>
        <td>one</td>
        <td>Same</td>
    </tr>
    
    <tr>
        <td>two</td>
        <td>Child</td>
    </tr>
    
</table>
<form action="/WORK/submit.task">
    <input type="submit" />
</form>
</body>
</html>